frontend/src/pages/Traces/TracePanel.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Dialog } from '@highlight-run/ui'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { TracePage } from '@/pages/Traces/TracePage'
+import { useParams } from '@/util/react-router/useParams'
+
+import * as styles from './TracePanel.css'
+
+export const TracePanel: React.FC = () => {
+	const navigate = useNavigate()
+	const { project_id } = useParams<{
```

Comment:
Not sure if there is a preference for `useParams` vs 
`const { projectId } = useProjectId()`?

Reply:
Yeah, should probably be `useProjectId` unless you need other params from the URL. Thanks for mentioning!

Reply:
I think I'm going to merge as is and handle this in a follow up!